backend/source/queries.py

```diff
@@ -0,0 +1,8 @@
+from graphene import ObjectType
+from graphene_django import DjangoListField
+
+from source.types.SourceType import SourceType
+
+
+class SourceQueries(ObjectType):
+    sources = DjangoListField(SourceType)
```

Comment:
DjangoListField is a handy util from `graphene_django` that has a built-in resolver. By default it returns all objects in the database. When we start to add filtering/sorting, we will have to write `resolve_sources()` ourselves.